<?xml version="1.0"?>

<launch>
  <arg name="input/object0" default="object0"/>
  <arg name="input/object1" default="object1"/>
  <arg name="output/object" default="merged_object"/>

  <node pkg="object_merger" type="object_association_merger_node" name="object_association_merger" output="screen">
    <remap from="~input/object0" to="$(arg input/object0)"/>
    <remap from="~input/object1" to="$(arg input/object1)"/>
    <remap from="~output/object" to="$(arg output/object)"/>
  </node>

</launch>
